nixos/modules/tasks/network-interfaces-scripted.nix

```diff
@@ -44,14 +44,16 @@ let
                          elem attrName deprecated && attr != null) bond);
   };
 
-  bondWarnings =
-    let oneBondWarnings = bondName: bond:
-          mapAttrsToList (bondText bondName) (bondDeprecation.filterDeprecated bond);
-        bondText = bondName: optName: _:
-          "${bondName}.${optName} is deprecated, use ${bondName}.driverOptions";
-    in {
-      warnings = flatten (mapAttrsToList oneBondWarnings cfg.bonds);
-    };
+  bondWarnings = {
+    warnings.networking.bonds.deprecatedOptions = lib.mapAttrs (bondName: bond:
+      lib.mergeAttrsList (map (optName: {
+        ${optName} = {
+          condition = (cfg.bonds.${bondName}.${optName} or null) != null;
```

Comment:
```suggestion
          condition = (cfg.bond.${optName} or null) != null;
```

Reply:
Actually after 7 years it's probably time to remove these legacy options and not waste any more time on them.

Reply:
Are you suggesting I remove the change, or do I just delete the deprecation warnings altogether?